smithery-mcp-server/src/services/document-ingestion.service.ts

```
@@ -0,0 +1,163 @@
+            {
+                target_server_id: 'firecrawl-mcp-server',
+                tool_name: 'scrape',
+                tool_args: {
+                    url: url,
+                    formats: ['markdown'],
+                    onlyMainContent: true,
+                }
+            }
+        );
+
+        if (toolboxResponse && toolboxResponse.content && toolboxResponse.content[0] && toolboxResponse.content[0].type === 'application/json') {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)